ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/Config.scala

```diff
@@ -40,7 +40,9 @@ object Config {
     ParticipantConfig.DefaultParticipantId -> ParticipantConfig()
   )
   val DefaultParticipantsDatasourceConfig: Map[ParticipantId, ParticipantDataSourceConfig] = Map(
-    ParticipantConfig.DefaultParticipantId -> ParticipantDataSourceConfig("default-jdbc-url")
+    ParticipantConfig.DefaultParticipantId -> ParticipantDataSourceConfig(
+      "jdbc:h2:mem:default;db_close_delay=-1;db_close_on_exit=false"
```

Comment:
Is it fine to have a default config with `db_close_on_exit=false`?

Reply:
I did not invent this one, it was there before just in another place, see ParticipantConfig in the current `main` branch